numpy/core/shape_base.pyi

```diff
@@ -26,6 +26,8 @@ def atleast_3d(__arys: ArrayLike) -> ndarray: ...
 @overload
 def atleast_3d(*arys: ArrayLike) -> List[ndarray]: ...
 
+def atleast_nd(ary: ArrayLike) -> ndarray: ...
```

Comment:
Note that this will also have to-be reexported to the main `numpy/__init__.pyi` file:
https://github.com/numpy/numpy/blob/f6a71feac4ac1623c8cd69dea5a560fe1e9af00d/numpy/__init__.pyi#L325-L333

Reply:
```suggestion
def atleast_nd(ary: ArrayLike, ndim: int, pos: int = ...) -> ndarray: ...
```
Also the `ndim` and `pos` parameters are missing.

Reply:
I really appreciate the help here. I understand that this is part of some larger type hinting mechanism, but I really have no idea what it's doing (same for the examples above)